resources/definitions/tenlog_printer.def.json

```
@@ -0,0 +1,295 @@
+{
+    "name": "tenlog_printer",
+    "version": 2,
+    "inherits": "fdmprinter",
+    "metadata": { 
+	"visible": false,
```

Mix of tabs and spaces

```suggestion
        "visible": false,
```